machAeroTutorials/aero_pygeo.rst

```diff
@@ -104,7 +104,14 @@ Run it yourself!
 After copying ``rae2822.dat`` to your ``aero/geometry`` folder and making the ``generate_wing.py`` script, you can now run the python file with the command
 
 .. prompt:: bash
-    
+
     python generate_wing.py
 
 You can open ``wing.dat`` in Tecplot to view the wing surface.
```

Comment:
Can you change this to say "You can open ``wing.dat`` in Tecplot. To view the wing surface, hide all zones called ``control_pts``." I would but don't have write access to this branch

Reply:
You should have push access now :)